frontend/src/metabase/redux/metadata.js

```diff
@@ -480,6 +497,17 @@ export const fetchDatabasesWithMetadata = createThunkAction(FETCH_DATABASES_WITH
     };
 });
 
+const FETCH_REAL_DATABASES_WITH_METADATA = "metabase/metadata/FETCH_REAL_DATABASES_WITH_METADATA";
+export const fetchRealDatabasesWithMetadata = createThunkAction(FETCH_REAL_DATABASES_WITH_METADATA, (reload = false) => {
```

Comment:
What does "real" databases mean?

Reply:
It won't include saved questions which are "virtual" databases. Will add a comment to that Redux action